<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SunsetPluginSettingsPaneClass</class>
 <widget class="QWidget" name="SunsetPluginSettingsPaneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SunsetPluginSettingsPane</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>&amp;Filename:</string>
       </property>
       <property name="buddy">
        <cstring>filenameInput</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="filenameInput">
         <property name="toolTip">
          <string>&lt;html&gt;The name of the data file to read the sunset times from. The filename can be absolute or relative. Relative filenames are searched in the plugin path.&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findFileButton">
         <property name="text">
          <string>&amp;Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>The filename can be specified without a directory
if the file is located in the plugin path or in the system path.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="SkLabel" name="filenameLabel">
       <property name="toolTip">
        <string>&lt;html&gt;The complete name of the data file&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string notr="true">[-]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Source:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="sourceLabel">
       <property name="toolTip">
        <string>&lt;html&gt;The source of the data, according to the data file&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string notr="true">[unknown]</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Reference longitude:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="referenceLongitudeLabel">
       <property name="toolTip">
        <string>&lt;html&gt;The longitude for which the times in the file are valid&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string notr="true">[unknown]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="longitudeLabel">
       <property name="text">
        <string>&amp;Longitude:</string>
       </property>
       <property name="buddy">
        <cstring>longitudeInput</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="longitudeCorrectionCheckbox">
       <property name="toolTip">
        <string>&lt;html&gt;Correct the sunset times for the actual longitude, specified below&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Enable correction</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="SkLabel" name="referenceLongitudeNoteLabel">
       <property name="text">
        <string>Longitude correction is only possible if
a reference longitude is specified in the data file.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="LongitudeInput" name="longitudeInput" native="true">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;The actual longitude&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="timeZoneLabel">
       <property name="text">
        <string>&amp;Timezone:</string>
       </property>
       <property name="buddy">
        <cstring>timeZoneInput</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="timeZoneInput">
         <property name="toolTip">
          <string>&lt;html&gt;Time zone for display. The time in the data file is always specified in UTC.&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>UTC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Local time</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SkLabel</class>
   <extends>QLabel</extends>
   <header>src/gui/widgets/SkLabel.h</header>
  </customwidget>
  <customwidget>
   <class>LongitudeInput</class>
   <extends>QWidget</extends>
   <header>src/gui/widgets/LongitudeInput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filenameInput</tabstop>
  <tabstop>findFileButton</tabstop>
  <tabstop>longitudeCorrectionCheckbox</tabstop>
  <tabstop>longitudeInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
